identity-account/src/account/account.rs

```diff
@@ -257,6 +258,30 @@ impl Account {
     Ok(())
   }
 
+  /// Fetches the latest changes for the account did from the tangle and
+  /// updates the local state
```

Comment:
```suggestion
  /// Fetches the latest changes from the tangle and **overwrites** the local document.
  /// 
  /// If a DID is managed from distributed accounts, this should be called before making changes
  /// to the identity, to avoid publishing updates that would be ignored.
```